src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/MessageIntegrationTest.kt

```diff
@@ -139,7 +139,7 @@ class MessageIntegrationTest : QueueIntegrationTest() {
       .expectBody()
       .jsonPath("index-status.currentIndex").isEqualTo(SyncIndex.INDEX_B.name)
       .jsonPath("index-status.inProgress").isEqualTo("true")
-      .jsonPath("index-size.${SyncIndex.INDEX_A.name}").isEqualTo("20")
+      .jsonPath("index-size.${SyncIndex.INDEX_A.name}").isEqualTo("23")
```

Comment:
I'm not keen on these tests for actual numbers that change every time more test data is added... I think it's better to just check that the result is > 0.